src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,49 @@
 package mate.academy;
 
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book book = new Book("Principles of Programming", new BigDecimal("100.00"));
+        System.out.printf("New book created: %s",
+                bookDao.create(book));
+        System.out.println("\n\nLet's try to get our book by ID...");
+        long id = book.getId();
+        Optional<Book> actual = bookDao.findById(id);
+        System.out.printf("Book got by id is: %s", actual.toString());
+
+        System.out.println("\n\nLet's try to change our book...");
+        book.setTitle("THE NEW Principles of Programming");
+        bookDao.update(book);
+        actual = bookDao.findById(id);
+        System.out.printf("Updated book is: %s", actual.toString());
```

Comment:
Similar to the previous comment, ensure that the `Optional` is not empty before calling `toString()`. Use `ifPresentOrElse` or check `actual.isPresent()` to handle the case where the book might not be found.